### DISCUSSION TITLE
[Locked] macos >10.15 is disabled?

### DISCUSSION THREAD

**Discussion by ghost** [actions/virtual-environments#2311] [Image configuration]
I'm create a run-os macos10.14 workflow,but,workflow run as macos10.15,i try macos10.13,workflow run as macos10.15 too.
macos >10.15 is disabled?

**Comment by maxim-lobanov** [comment]
On GitHub Actions, only `macos-10.15` and `macos-11.0` are available.
On Azure DevOps, only `macos-10.14` and `macos-10.15` are available.
